queue_services/entity-bn/src/entity_bn/config.py

```diff
@@ -100,6 +100,10 @@ class _Config():  # pylint: disable=too-few-public-methods
     EMAIL_PUBLISH_OPTIONS = {
         'subject': os.getenv('NATS_EMAILER_SUBJECT', 'entity.email'),
     }
+    
+    ENTITY_PUBLISH_OPTIONS = {
+        'subject': os.getenv('NATS_ENTITY_EVENTS_SUBJECT', 'entity.events'),
+    }
```

Comment:
You can reuse the subject from SUBSCRIPTION_OPTIONS (line: 95). If you prefer to keep publish option separate like this change `NATS_ENTITY_EVENTS_SUBJECT` to `NATS_ENTITY_EVENT_SUBJECT`

Reply:
sounds good I just reused the subscription options